nalgebra-sparse/src/csc.rs

```diff
@@ -792,3 +796,35 @@ where
         self.lane_iter.next().map(|lane| CscColMut { lane })
     }
 }
+
+/// Iterator for [CscCol](struct.CscCol.html).
```

Comment:
You can directly use [Rustdoc intra-doc links](https://doc.rust-lang.org/nightly/rustdoc/write-documentation/linking-to-items-by-name.html) here, which is preferred to directly writing the URL. In this case I believe it would be simply ``[`CscCol`]``, although please test this by rendering the docs with `cargo test`.

The same applies to the other doc comments with doc links!